samples/MegaDrone/src/main/cpp/MonoToStereo.h

```diff
@@ -36,8 +35,8 @@ class MonoToStereo : public RenderableAudio<T> {
         // e.g. 123 => 112233
         for (int i = numFrames - 1; i >= 0; --i) {
 
-            audioData[i * kStereoChannelCount] = audioData[i];
-            audioData[i * kStereoChannelCount + 1] = audioData[i];
+            audioData[i * oboe::ChannelCount::Stereo] = audioData[i];
```

Comment:
This used to be an int. Now it is an enum. Doing math on enums is a bit sketchy.

Reply:
Nice CL otherwise. Thanks.

Reply:
ok, let me change the underlying to int32_t type